content/applications/inventory_and_mrp/inventory/management/shipments_deliveries/delivery_three_steps.rst

```diff
@@ -28,27 +28,27 @@ three step deliveries. First, make sure the *Multi-Step Routes* option is enable
    :alt: Activate multi-step routes and storage locations in inventory settings.
 
 Next, the warehouse needs to be configured for three step deliveries. To do this, go to
-:menuselection:`Inventory --> Configuration --> Warehouses`, and click :guilabel:`Edit` to edit the
-:guilabel:`Warehouse`. Then, select :guilabel:`Pack goods, send goods in output and then deliver
-(3 steps)` for :guilabel:`Outgoing Shipments`. Then :guilabel:`Save` the changes.
+:menuselection:`Inventory --> Configuration --> Warehouses`, and click on the warehouse to edit.
+Then, select :guilabel:`Pack goods, send goods in output and then deliver (3 steps)` for
+:guilabel:`Outgoing Shipments`.
 
 .. image:: delivery_three_steps/three-step-warehouse-config.png
    :align: center
    :alt: Set outgoing shipment option to deliver in three steps.
 
 Activating three-step receipts and deliveries will create two new internal locations: a
 *Packing Zone* (WH/Packing Zone), and *Output* (WH/Output). To rename these locations, go to
-:menuselection:`Configuration --> Locations`, select the :guilabel:`Location` to change, click
-:guilabel:`Edit`, update the name, and finally, click :guilabel:`Save`.
+:menuselection:`Configuration --> Locations`, click on the :guilabel:`Location` to change, and
+update the name.
 
 Process a delivery order in three steps (pick + pack + ship)
```

Comment:
I changed this to "Deliver in three steps (pick + pack + ship)" because that is how it appears in Odoo under the routes section. So I kept the '+'s' since that also is how it appears (this way, for people following along and looking around it'll be a perfect match).